addons/vote.py

```diff
@@ -45,43 +45,51 @@ def __init__(self, bot):
             json.dump(self.vote_list, votefile)
 
     @commands.guild_only()
-    @commands.command()
-    async def poll(self, ctx, command, link="", name="", *, options=""):
-        if command == "create" or command == "start":
-            if not self.poll_ongoing:
-                if not link or not name or not options:
-                    return await ctx.send(f"You didn't provide an argument. You should fix that. Inputted: link-{link}, name-{name}, options-{options}.")
-                vote_options = []
-                self.pollcfg["name"] = name
-                self.pollcfg["link"] = link
-                for option in options.split(" | "):
-                    vote_options.append(option)
-                self.pollcfg["options"] = vote_options
-                with open("poll.json", "w") as pollfile:
-                    json.dump(self.pollcfg, pollfile)
-                with open(f"{name}_votes.json","w") as votefile:
-                    json.dump(self.vote_list, votefile)
-                self.poll_ongoing = True
-                await ctx.send("Poll successfully created!")
-            else:
-                await ctx.send("There is poll ongoing already!")
-                
-        if command == "close" or command == "end":
-            if self.poll_ongoing:
-                # prob add evi final tally before finishing poll
-                os.makedirs("Polls", exist_ok=True)
-                with open("Polls/{}.json".format(self.pollcfg["name"]), "w") as pollbackup:
-                    json.dump(self.pollcfg, pollbackup)
-                with open("Polls/{}_votes.json".format(self.pollcfg["name"]), "w") as votesbackup:
-                    json.dump(self.vote_list, votesbackup)
-                self.pollcfg = {}
-                self.vote_list = {}
-                with open("poll.json", "w") as pollfile:
-                    json.dump(self.pollcfg, pollfile)
-                self.poll_ongoing = False
-                await ctx.send("Poll successfully closed!")
-            else:
-                await ctx.send("There is no ongoing poll!")
+    @commands.group(pass_context=True)
+    async def poll(self, ctx):
+        if ctx.invoked_subcommand is None:
+            await ctx.send('Invalid command: {0.subcommand_passed}'.format(ctx))
```

Comment:
It seems like you're merging f-strings and formatting here? Also please stick to full quotes across the file, not single quotes.

Reply:
Fixed in my next commit